fury/actor.py

```diff
@@ -754,6 +754,80 @@ def callback(_caller, _event, calldata=None):
     return actor
 
 
+def dashed_line(p_initial =[0.0, 0.0, 0.0], p_final=[0.0, 0.0, 0.0], 
+                point_size=10, num_points=10, point_color='Peacock'):
+    """Create an actor for one or more dashed lines.
+
+    Parameters
+    ------------
+    p_initial : list(1, 3)
+        Takes start point. Default is [0.0, 0.0, 0.0]. 
+    p_final : list(1, 3)
+        Takes end point. Default is [0.0, 0.0, 0.0]. 
+    point_size : float  
+        Takes size of point.
+    num_points : int
+        Takes number of points in the dashed line.
+    point_color : string
+        Takes color of point.
+
+    Returns
+    ----------
+    v : vtkActor 
+        Dashed Line.
+
+    Examples
+    ----------
+    >>> from fury import actor, window
+    >>> scene = window.Scene()
+    >>> p_initial   = [1.0, 0.0, 0.0]
+    >>> p_final     = [0.0, 1.0, 0.0]
+    >>> point_size  = 5
+    >>> num_points  = 20
+    >>> point_color = 'Peacock'
+    >>> c = actor.dashed_line(p_initial, p_final, 
+    >>> point_size, num_points, point_color)
+    >>> scene.add(c)
+    >>> #window.show(scene)
+
+    """
+    colors = vtk.vtkNamedColors()
+    
+    #Convert p_initial and p_final to arrays
+    p_initial = np.array(p_initial)
+    p_final = np.array(p_final)
+    dp = (p_final - p_initial)/num_points
+    p = p_initial.copy()
+
+    mapper = vtk.vtkPolyDataMapper()
+    points = vtk.vtkPoints()
+    count = 0
+    pid = [0] * num_points
+
+    #Add each point ID
+    while (count < num_points):
+        pid[count] = points.InsertNextPoint(list(p))
+        p += dp
+        count += 1
+
+    vertices = vtk.vtkCellArray()
+    vertices.InsertNextCell(num_points, pid)
+
+    # Create a polydata object
+    point = vtk.vtkPolyData()
+
+    # Set the points and vertices as the geometry/topology of the polydata
+    point.SetPoints(points)
+    point.SetVerts(vertices)
+    mapper.SetInputData(point)
+
+    actor = vtk.vtkActor()
```

Comment:
We have a `point` actor and `line` actor. Can you use them instead of going from scratch? Thank you

Reply:
Sure, I will try to implement it using the suggested inbuilt actors.
Thank you very much for reviewing.